Detectors/TPC/workflow/include/TPCWorkflow/CATrackerSpec.h

```diff
@@ -60,6 +61,9 @@ struct Config {
       case Operation::ZSDecoder:
         zsDecoder = true;
         break;
+      case Operation::ZSOnTheFly:
+    	zsOnTheFly = true;
```

Comment:
I thought you ran git-clang-format? I would propose to do it before every commit.

Reply:
indeed, my git hook did not work :-/ 
should be fixed and not happen again